trackier-ios-sdk/Classes/TrackierEvent.swift

```diff
@@ -28,6 +28,8 @@ public class TrackierEvent {
     public var param10: String? = ""
 
     var revenue: Float64? = 0.0
+    public var setDiscount: Float64? = 0.0
+    public var setCouponCode: String? = ""
```

Comment:
variable name can not be setSomething, plz fix this @IloveJavaa 

create getter/setters for this

Reply:
Changes done

<img width="437" alt="Screenshot 2022-08-08 at 11 22 56 AM" src="https://user-images.githubusercontent.com/16884982/183348651-41cae5d0-d5e5-4431-808c-122080e0116f.png">